pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -198,14 +269,50 @@ def main(args):
                                           merge_with=config_lines(args))
     science_path = os.path.join(parset['rdx']['redux_path'], parset['rdx']['scidir'])
 
-    # Calibration Master directory
-    if args.master_dir is None:
-        msgs.error("You need to set an Environmental variable MOSFIRE_MASTERS that points at the Master Calibs")
+    # Parse the files sort by MJD
+    files = np.array([os.path.join(args.full_rawpath, file) for file in args.files])
+    nfiles = len(files)
+    mjds = np.zeros(nfiles)
```

Comment:
Use `metadata.py` class/routines for this, e.g. `PypeItMetaData`

Reply:
Well I don't plan to a a depdendence on PypeItMetaData here, since I don't need to. You might argue I should build the fitstable, but that would add a dependency on 1700 lines of code, which often fails for situations where files don't have certain header cards because instrument controller had problems etc. This happens just about once per night for e.g. MOSFIRE. This is hardly necessary to get the MJD which I only need to sort the output, it will stay as is unless fitstable is rewritten to never crash.

Reply:
Add several TODO's that we need to refactor this code
to (1) generate a simple PypeIt file and (2) run that.

As in the under-development `ql_keck_demos` script.